packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/__snapshots__/ListItem.spec.ts.snap

```diff
@@ -1,17 +1,17 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
 exports[`Collaborator ListItem component > share inheritance indicators > show when sharedParentRoute is given 1`] = `
-"<div data-v-ccf14be2="" data-testid="collaborator-user-item-brian" class="files-collaborators-collaborator oc-py-xs">
+"<div data-v-ccf14be2="" data-testid="collaborator-user-item-einstein" class="files-collaborators-collaborator oc-py-xs">
   <div data-v-ccf14be2="" class="oc-width-1-1 oc-flex oc-flex-middle files-collaborators-collaborator-details">
     <div data-v-ccf14be2="" class="oc-width-2-3 oc-flex oc-flex-middle">
       <div data-v-ccf14be2="">
-        <avatar-image-stub data-v-ccf14be2="" userid="brian" user-name="Brian Murphy" width="36" class="files-collaborators-collaborator-indicator"></avatar-image-stub>
+        <avatar-image-stub data-v-ccf14be2="" userid="3" user-name="einstein" width="36" class="files-collaborators-collaborator-indicator"></avatar-image-stub>
       </div>
       <div data-v-ccf14be2="" class="oc-pl-s oc-text-truncate">
-        <div data-v-ccf14be2="" class="oc-text-truncate"><span data-v-ccf14be2="" aria-hidden="true" class="files-collaborators-collaborator-name">Brian Murphy</span> <span data-v-ccf14be2="" class="oc-invisible-sr">Share receiver name: Brian Murphy (brian@owncloud.com)</span></div>
+        <div data-v-ccf14be2="" class="oc-text-truncate"><span data-v-ccf14be2="" aria-hidden="true" class="files-collaborators-collaborator-name">einstein</span> <span data-v-ccf14be2="" class="oc-invisible-sr">Share receiver name: einstein</span></div>
```

Comment:
Is it desired that the full name plus email was replaced with just the username here?